Makefile

```diff
@@ -74,6 +74,23 @@ ifeq (, $(SHELLCHECK_COMMAND))
 endif
 SHELL_FILES := $(shell find $(MAKEFILE_DIR) -type f -name '*.sh')
 
+.PHONY: lint
+transform-al2-to-al2023-config:
+	cat $(PACKER_TEMPLATE_FILE) | \
+	jq '._comment = "All template variables are enumerated here; and most variables have a default value defined in eks-worker-al2023-variables.json"' | \
+	jq '.variables.temporary_key_pair_type = "ed25519"' \
+	> $(subst al2,al2023,$(PACKER_TEMPLATE_FILE))
+
+	# TODO: switch to minimal "al2023-ami-minimal-2023.*-kernel-6.1-x86_64"
```

Comment:
Why not start out with minimal? It likely will work out-of-the-box for this use case.

Reply:
figured out why minimal was not working with `instance connect`, now that i know we need `sudo yum install ec2-instance-connect` will switch to minimal image

Reply:
Done https://github.com/kubernetes-sigs/provider-aws-test-infra/pull/73